<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BankAssignmentMatrix</class>
 <widget class="QDialog" name="BankAssignmentMatrix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoundBank自动分配工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnAddRow">
       <property name="text">
        <string>添加行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveRow">
       <property name="text">
        <string>删除行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAddColumn">
       <property name="text">
        <string>添加列</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveColumn">
       <property name="text">
        <string>删除列</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tblMatrix">
     <row>
      <property name="text">
       <string>1</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>2</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>3</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>4</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>5</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnFillWithChildren">
       <property name="text">
        <string>使用选取对象填充行</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreateBanks">
       <property name="text">
        <string>创建新的SoundBank</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAssignMedia">
       <property name="text">
        <string>分配到已有SoundBank</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
